src/pages/ManagerHomepage/ManagerHomepage.tsx

```diff
@@ -0,0 +1,13 @@
+import { BookCatalogueSection } from '../Homepage/BookCatalogueSection'
+import { LetsStartExploring } from '../Homepage/LetsStartExploring'
+
+const Library = () => {
```

Comment:
This page needs to have same layout as Homepage, you can reuse CSS from Homepage.module.css